core/test/functional/api/v2/admin/configuration_spec.js

```diff
@@ -0,0 +1,82 @@
+var should = require('should'),
+    supertest = require('supertest'),
+    testUtils = require('../../../../utils'),
+    localUtils = require('./utils'),
+    config = require('../../../../../../core/server/config'),
+    ghost = testUtils.startGhost,
+    request;
+
+describe('Configuration API', function () {
+    var accesstoken = '', ghostServer;
```

Comment:
These two variables can be removed as they are never used anymore. Not sure this is a part of migration but I'm doing this cleanup for the tests that are being copied :)